src/components/events/Modal.jsx

```diff
@@ -1,3 +1,35 @@
-const Modal = () => {};
+import { IoMdClose } from "react-icons/io";
+
+const Bullet = ({ item }) => (
+  <li className="flex items-center mb-1">
+    <span className="w-3 h-3 bg-biscuits-blue-100 inline-block rounded-full mr-2"></span>
+    {item}
+  </li>
+);
+
+const Modal = ({ title, date, time, place, description }) => {
```

Comment:
```suggestion
const Modal = ({ Title="Event title", date="Date", time="Time", place="Place", description="no description provided." }) => {
```
input default items directly